variants.yaml

```diff
@@ -0,0 +1,9 @@
+variants:
+  cloud-controller-manager-amd64:
+    CLOUD_BUILD_IMAGE: ccm-amd64
+  cloud-controller-manager-arm:
+    CLOUD_BUILD_IMAGE: ccm-arm
+  cloud-controller-manager-arm64:
+    CLOUD_BUILD_IMAGE: ccm-arm64
```

Comment:
would this build 3 ccm images or still merged into one multi-arch image?

Reply:
Until now, ccm images are not merged into 1 multi-arch image and I didn't touch the script. Do you suggest doing so, like cnm?
https://console.cloud.google.com/gcr/images/k8s-staging-provider-azure/GLOBAL

Reply:
Discussed offline. Let's make ccm images a multi-arch image.